workitem/workitem_repository_blackbox_test.go

```diff
@@ -257,7 +256,7 @@ func (s *workItemRepoBlackBoxTest) TestGetCountsPerIteration() {
 	iteration2 := iteration.Iteration{
 		Name:       "Sprint 2",
 		SpaceID:    spaceInstance.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c